### ISSUE TITLE
NotFoundError (see above for traceback): Key fc3/BatchNorm/moving_mean not found in checkpoint

### ISSUE THREAD

**Issue by wqzdhrdhr** [hwalsuklee/tensorflow-mnist-cnn#9]
Thanks for your code! And I have some questions, glad for your help.
With the Ubuntu 18.04 and the tensorflow 0.12.0, when I run 
`python mnist_cnn_test.py --model-dir model/model01_99.61 --batch-size 5000 --use-ensemble False`
It don't work and return `NotFoundError (see above for traceback): Key fc3/BatchNorm/moving_mean not found in checkpoint
	 [[Node: save/RestoreV2_9 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/cpu:0"](_recv_save/Const_0, save/RestoreV2_9/tensor_names, save/RestoreV2_9/shape_and_slices)]]`

And when I run 'python mnist_cnn_test.py --model-dir model/model01_99.61 --batch-size 5000 --use-ensemble True'
The "accuracy“ only 0.092, All models are like this.
Looking forward to your reply.
